aws-amp/adot/federated-store.yaml

```diff
@@ -0,0 +1,5 @@
+type: S3
```

Comment:
This is paired down pretty heavily from our standard example. Is there any concern with leaving out the other values?

Reply:
it is what I use.

We aren't Thanos, but this is what they say:
At a minimum, you will need to provide a value for the bucket, endpoint, access_key, and secret_key keys. The rest of the keys are optional.
https://thanos.io/tip/thanos/storage.md/